### PR TITLE
Groom changelog /1.5.1 beta.2

### PR DESCRIPTION
**Pull request by mgamis-msft**
# What

# Why

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/communication-react/CHANGELOG.beta.md [hunk 1]
```diff
@@ -9,37 +9,42 @@ This log was last generated on Tue, 28 Mar 2023 16:00:31 GMT and should not be m
 Tue, 28 Mar 2023 16:00:31 GMT 
 [Compare changes](https://github.com/azure/communication-ui-library/compare/@azure/communication-react_v1.5.1-beta.1...@azure/communication-react_v1.5.1-beta.2)
 
-### Minor changes
+### Vertical Gallery - Public Preview
+We are now introducing new overflow gallery experiences to our VideoGallery component!
+
+- These new experiences include the new vertical experience. This new overflow gallery will allow developers to better utilize the vertical space in their calling experiences
```
Comment (edwardlee-msft) [inline packages/communication-react/CHANGELOG.beta.md:15]: nit: the use of 'experience' three times in these two sentences feels quite off .

File: packages/communication-react/CHANGELOG.beta.md [hunk 2]
```diff
@@ -9,37 +9,42 @@ This log was last generated on Tue, 28 Mar 2023 16:00:31 GMT and should not be m
...
+- The horizontal experience is also seeing improvements to space utilized by its video tiles
```
Comment (edwardlee-msft) [inline packages/communication-react/CHANGELOG.beta.md:16]: nit: - improvements to space utilized by the video tiles in Horizontal Gallery

File: packages/communication-react/CHANGELOG.beta.md [hunk 3]
```diff
@@ -9,37 +9,42 @@ This log was last generated on Tue, 28 Mar 2023 16:00:31 GMT and should not be m
...
+- Also in these new experiences are enhancements to our dominant speaker ordering and video tile rendering in the overflow galleries.
```
Comment (edwardlee-msft) [inline packages/communication-react/CHANGELOG.beta.md:17]: - Enhancements to dominant speaker ordering and video tile rendering in the overflow gallery

File: packages/communication-react/CHANGELOG.beta.md [hunk 4]
```diff
@@ -9,37 +9,42 @@ This log was last generated on Tue, 28 Mar 2023 16:00:31 GMT and should not be m
...
+  - VideoGallery in call composite will use a vertical overflow gallery when its aspect ratio is 16:9 or greater. ([PR #2786](https://github.com/azure/communication-ui-library/pull/2786) by 79475487+mgamis-msft@users.noreply.github.com)
+  - Update overflow gallery logic to allow more video participants to render video streams if available on different pages. ([PR #2818](https://github.com/azure/communication-ui-library/pull/2818) by 94866715+dmceachernmsft@users.noreply.github.com)
+  - Create styles for verticalGallery controls and position. ([PR #2781](https://github.com/azure/communication-ui-library/pull/2781) by 94866715+dmceachernmsft@users.noreply.github.com)
 
-### Patches
+### Bug fixes
 
 - `@azure/communication-react`
   - disable the selectivity of a text in video gallery for long touch to function properly ([PR #2790](https://github.com/azure/communication-ui-library/pull/2790) by 97124699+prabhjot-msft@users.noreply.github.com)
   - Update storybook custom data model calling example to wrap userId as an identifier ([PR #2799](https://github.com/azure/communication-ui-library/pull/2799) by edwardlee@microsoft.com)
```
Comment (edwardlee-msft) [inline packages/communication-react/CHANGELOG.beta.md:33]: - can remove storybook and internal changes.

File: packages/communication-react/CHANGELOG.beta.md [hunk 5]
```diff
@@ -9,37 +9,42 @@ This log was last generated on Tue, 28 Mar 2023 16:00:31 GMT and should not be m
...
-  - Fixed richtext css for blockquote and table ([PR #2839](https://github.com/azure/communication-ui-library/pull/2839) by 77021369+jimchou-dev@users.noreply.github.com)
   - Fix CallAdapter participants joined and left events to show correct participants in event array. ([PR #2837](https://github.com/azure/communication-ui-library/pull/2837) by 94866715+dmceachernmsft@users.noreply.github.com)
   - Correct MessageThread API definition ([PR #2812](https://github.com/azure/communication-ui-library/pull/2812) by 3941071+emlynmac@users.noreply.github.com)
-  - Update internal dispose view logic to be under one utility function for remote and local streams. ([PR #2758](https://github.com/azure/communication-ui-library/pull/2758) by 94866715+dmceachernmsft@users.noreply.github.com)
   - Fix dominant speaker ordering to best keep dominant speakers on the first page of overflow gallery. ([PR #2819](https://github.com/azure/communication-ui-library/pull/2819) by miguelgamis@microsoft.com)
   - make sure message status in storybook is defined to avoid 'try send again' showing up ([PR #2732](https://github.com/azure/communication-ui-library/pull/2732) by carolinecao@microsoft.com)
```
Comment (edwardlee-msft) [inline packages/communication-react/CHANGELOG.beta.md:47]: Some of these have periods at the end some of them don't. 
Can we be consistent with the formatting?

Also, capitalization here as well.

File: packages/communication-react/CHANGELOG.beta.md [hunk 6]
```diff
@@ -9,37 +9,42 @@ This log was last generated on Tue, 28 Mar 2023 16:00:31 GMT and should not be m
...
-  - Update code owners ([PR #2737](https://github.com/azure/communication-ui-library/pull/2737) by joshlai@microsoft.com)
-  - Update overflow gallery logic to allow more video participants to render video streams if available on different pages. ([PR #2818](https://github.com/azure/communication-ui-library/pull/2818) by 94866715+dmceachernmsft@users.noreply.github.com)
   - Announce by Screen Reader after leaving chat ([PR #2681](https://github.com/azure/communication-ui-library/pull/2681) by 107075081+Leah-Xia-Microsoft@users.noreply.github.com)
   - Added empty icon render for tab navigation of video tile ([PR #2693](https://github.com/azure/communication-ui-library/pull/2693) by 97124699+prabhjot-msft@users.noreply.github.com)
   - Split buttons now hold primary action for toggling camera and mic on touch devices. ([PR #2773](https://github.com/azure/communication-ui-library/pull/2773) by 94866715+dmceachernmsft@users.noreply.github.com)
-  - Fixed richtext css for blockquote and table ([PR #2839](https://github.com/azure/communication-ui-library/pull/2839) by 77021369+jimchou-dev@users.noreply.github.com)
```
Comment (edwardlee-msft) [inline packages/communication-react/CHANGELOG.beta.md:36]: Some of these are past tense and some or present tense. 
Can we be consistent and use only one?

File: packages/communication-react/CHANGELOG.beta.md [hunk 7]
```diff
@@ -80,13 +83,14 @@ Tue, 28 Mar 2023 16:00:31 GMT
...
   - Updates LocalVideo tile to be 9:16 aspect ratio to show whole feed. fixes CallWithChat flashing issue with scrollable gallery. ([PR #2734](https://github.com/azure/communication-ui-library/pull/2734) by 94866715+dmceachernmsft@users.noreply.github.com)
   - Draft API for video background effects ([PR #2836](https://github.com/azure/communication-ui-library/pull/2836) by 97124699+prabhjot-msft@users.noreply.github.com)
-  - VideoGallery in call composite will use a vertical overflow gallery when its aspect ratio is 16:9 or greater. ([PR #2786](https://github.com/azure/communication-ui-library/pull/2786) by 79475487+mgamis-msft@users.noreply.github.com)
   - Add onFetchProfile to adapter ([PR #2680](https://github.com/azure/communication-ui-library/pull/2680) by jiangnanhello@live.com)
   - Set all empty/undefined displayName to unnamed ([PR #2720](https://github.com/azure/communication-ui-library/pull/2720) by jinan@microsoft.com)
+  - Add 'Try ChatComposite' page to storybook that uses fake adapters ([PR #2710](https://github.com/azure/communication-ui-library/pull/2710) by 77021369+jimchou-dev@users.noreply.github.com)
```
Comment (edwardlee-msft) [inline packages/communication-react/CHANGELOG.beta.md:91]: storybook change can be most likely removed

File: packages/communication-react/CHANGELOG.beta.md (hunk 1 above)
Comment (dmceachernmsft) [reply packages/communication-react/CHANGELOG.beta.md:15]: 😭

**Review by edwardlee-msft — APPROVED** [review]

File: packages/communication-react/CHANGELOG.beta.md (hunk 6 above)
Comment (mgamis-msft) [reply packages/communication-react/CHANGELOG.beta.md:36]: Changed all to present tense

File: packages/communication-react/CHANGELOG.beta.md (hunk 5 above)
Comment (mgamis-msft) [reply packages/communication-react/CHANGELOG.beta.md]: No more periods :)

**Pull request merged**